### PR TITLE
i18n(es): update `environmental-impact`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- Closes #  N/A
- What does this PR change? Give us a brief description.
Updates `enviromental-impact` based on this commit c71b0a24c26d123f2f80fdc32bbc1343470a3541

### CODE REVIEW

File: docs/src/content/docs/es/environmental-impact.md [hunk 1]
```diff
@@ -73,20 +73,21 @@ Elegir un proveedor de alojamiento que utilice energía renovable significa tene
...
-| [Sphinx][sx-carbon]         | 0.07g                      |
-| [MkDocs][mk-carbon]         | 0.10g                      |
-| [Nextra][nx-carbon]         | 0.11g                      |
-| [docsify][dy-carbon]        | 0.11g                      |
-| [Docusaurus][ds-carbon]     | 0.24g                      |
-| [Read the Docs][rtd-carbon] | 0.24g                      |
-| [GitBook][gb-carbon]        | 0.71g                      |
-
-<small>Datos recopilados el 14 de mayo de 2023. Haz clic en un enlace para ver cifras actualizadas.</small>
+| Framework                   | CO₂ per page visit | Rating |
```
Comment (delucis) [inline docs/src/content/docs/es/environmental-impact.md:76]: Should the headings here be translated?

“CO₂ per page visit” was “CO₂ por visita a la página” originally and I guess there’s a translation for “Rating” too?

**Review by delucis — CHANGES_REQUESTED** [review]
Thanks @dreyfus92! One question about the table heading.

File: docs/src/content/docs/es/environmental-impact.md (hunk 1 above)
Comment (dreyfus92) [reply docs/src/content/docs/es/environmental-impact.md:76]: I forgot about those, let me fix it real quick 😁

Comment (dreyfus92) [inline docs/src/content/docs/es/environmental-impact.md:76]:
```suggestion
| Framework                   | CO₂ por visita a la página | Calificación |
```

Comment (delucis) [reply docs/src/content/docs/es/environmental-impact.md:76]: LGTM!

**Review by kevinzunigacuellar — APPROVED** [review]
🚢

**Review by delucis — APPROVED** [review]

**Pull request merged**